### ISSUE TITLE
[EN] [Dungeon] Stuck Casters: Ragefire - Gnomeregan

### ISSUE THREAD

**Issue by Tj20** [Atlantiss/BugTracker#1697]
This is a list of specific issues, related to the general issue[#1018](https://github.com/Atlantiss/BugTracker/issues/1018).

Wolfenstein said [here](http://atlantiss.eu/community/viewtopic.php?f=22&t=637) that stuck casters are easy to fix, but it has to be done for every mob type affected, if I understood correctly.

So I thought: maybe a list of affected mobs in dungeons and the ability they are trying to cast without success might be helpful:

### [Ragefire Chasm](http://www.wowhead.com/zone=2437/ragefire-chasm)
- [Ragefire Shaman](http://www.wowhead.com/npc=11319/ragefire-shaman) - [Lightning Bolt](http://www.wowhead.com/spell=9532)
- [Molten Elemental](http://www.wowhead.com/npc=11321/molten-elemental) - although a melee mob, it gets stuck on casting *Fire Shield*
- [Searing Blade Cultist](http://www.wowhead.com/npc=11322/searing-blade-cultist) - Shadow Bolt

### [Deadmines](http://www.wowhead.com/zone=1581/the-deadmines)
- [Defias Squallshaper](http://www.wowhead.com/npc=48521/defias-squallshaper) - they follow until they are <40yd, but cannot cast [Seaswell](http://www.wowhead.com/spell=90398) until they are <30yd, so they have a "deadzone"

### [Wailing Caverns](http://www.wowhead.com/zone=718/wailing-caverns)
- [Druid of the Fang](http://www.wowhead.com/npc=3840/druid-of-the-fang) - [Lightning Bolt](http://www.wowhead.com/spell=9532) (sometimes they are also stuck after their transformation although they should just melee in snake form)

### [Shadowfang Keep](http://www.wowhead.com/zone=209/shadowfang-keep)
- [Dread Scryer](http://www.wowhead.com/npc=47141/dread-scryer) - they only have instant [Arcane Missiles](http://www.wowhead.com/spell=33833)/[Cone of Cold](http://www.wowhead.com/spell=12611) but still get stuck

### [Blackfathom Deeps](http://www.wowhead.com/zone=719/blackfathom-deeps)
- [Blackfathom Sea Witch](http://cata.openwow.com/npc=4805) - Blizzard/[Frost Nova](http://cata.openwow.com/spell=122) don't take much cast time, but she still doesn't move, even when casting nothing. She just melees if you are near her.
- [Fallenroot Hellcaller](http://cata.openwow.com/npc=4799) - [Shadow Bolt](http://cata.openwow.com/spell=9613)
- [Blindlight Oracle](http://cata.openwow.com/npc=4820) - [Lightning Bolt](http://cata.openwow.com/spell=9532)
- [Twilight Aquamancer](http://cata.openwow.com/npc=4811) - [Frostbolt](http://cata.openwow.com/spell=9672)

### [The Stockade](http://www.wowhead.com/zone=717/the-stockade)
- [Riverpaw Shaman](http://www.wowhead.com/npc=46252/riverpaw-shaman) - [Lightning Bolt](http://www.wowhead.com/spell=12167)

### [Gnomeregan](http://www.wowhead.com/zone=721/gnomeregan)
- [Leprous Defender](http://www.wowhead.com/npc=6223) - [Shoot](http://www.wowhead.com/spell=6660)
- [Leprous Technician](http://www.wowhead.com/npc=6222/leprous-technician) - [Throw Wrench](http://www.wowhead.com/spell=13398)
- [Leprous Machinesmith](http://www.wowhead.com/npc=6224/leprous-machinesmith) - [Throw Wrench](http://www.wowhead.com/spell=13398)

If you think this isn't really helpful or if I should use a different format, let me know. I thought I put a couple of affected mobs into 1 bug report, since I didn't want to spam the bug tracker with lots of tiny issues (apparently they have to change only 1 flag per mob type).

**Comment by Varchord** [comment]
I think will add some mobs i remember from TBC dungeons during the weekend

**Comment by Tj20** [comment]
I was originally thinking of doing a seperate issue for the next couple of dungeons so one issue doesn't get too long and could be closed more easily.
But on the other hand it might be easier/more clear to collect things in one list first.

So yeah, please do that. I will edit this issue and we can collect stuff here. The devs can tell us if they prefer a different format or something, I guess. :)

The dungeons I testest so far, I did run in, aggro every mob, run 40+ yd and had a look which ones were stuck to their places. 

I think I can do a few more, but then my level will be too low to solo them (and groups probably would just like to kill stuff. not let all the casters live to see how they behave).

**Comment by Tj20** [comment]
yeah... I'm done here. My level is too low to solo any more dungeons. :D

**Comment by sunshineinabox** [comment]
Casters in BRC also get "stuck" in their walking path so even after you attack them they are not agroed they just keep walking and casting.

**Comment by karama300** [comment]
Great work pulled here but no result yet. :(

**Comment by Fapguy** [comment]
This needs more attention, it's one of those "qualities of life" things.

**Comment by Cebarus** [comment]
+1 it's been 2 years don't let this ruin the immersion
